<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(86, 86, 86);</string>
  </property>
  <widget class="QLabel" name="label_menu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Statue : ++</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 136, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_Etudiant">
    <attribute name="title">
     <string>salarie</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Cordonnées</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="id">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nom </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="texte_Nom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>salaire</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="salaire"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>heure abs</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="abs"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>tache realiser </string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tache"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 127);
font: 900 9pt &quot;Verdana Pro Cond Black&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 127);
font: 900 9pt &quot;Verdana Pro Cond Black&quot;;</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_modifier">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 127);
font: 900 9pt &quot;Verdana Pro Cond Black&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_afficher">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 0, 127);
font: 900 9pt &quot;Verdana Pro Cond Black&quot;;</string>
        </property>
        <property name="text">
         <string>Afficher </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Professeur">
    <attribute name="title">
     <string>fournisseur</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>561</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_ajouter_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 0);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_modifier_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_supprimer_2">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_afficher_2">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Afficher </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>561</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Cordonnées</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idf">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nom </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nomf"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>adress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="adress"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Téléphone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tele"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mail"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Matiere">
    <attribute name="title">
     <string>produit</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>551</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QPushButton" name="pushButton_ajouter_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_modifier_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_supprimer_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_afficher_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Afficher </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Cordonnées</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="idp">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nomp"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>prix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="prix"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>qte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qte"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>fournisseur </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fournisseur"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>1071</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Picture">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
